userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -57,14 +57,15 @@ constexpr const cgroup_layout CRI_CGROUP_LAYOUT[] = {
 
 cri::cri(container_cache_interface &cache) : container_engine_base(cache)
 {
-	if (s_cri_unix_socket_paths.empty())
+	libsinsp::cri::settings* cri_settings = libsinsp::cri::settings::instance();
+	if (cri_settings->get_cri_unix_socket_paths().empty())
```

Comment:
Could we add just a tiny new test to make sure the default invocation adds all default socket paths (and no one breaks it in the future)? The new logic would then also go through the LeakSanitizer test in our CI.